cfpb/college-costs
# Gradplus Loan visibility, undergrad property now derived from program

The page will now show or hide "Direct PLUS" (aka GradPLUS) loans depending on program level. This will also update the `undergrad` property, which is used by `student-debt-calc`.
## Additions
- a `div` for gradplus, with the property `data-section="gradPlus"`
- Function to show or hide the `gradPlus` section
- `updateModelWithProgram` method in `financial-model.js`, which updates the financial model with properties derived from school and program data
- functional test for gradplus visibility
## Changes
- Changed API calls to separate `iped` and `pid` with an underscore
## Testing
- Pull down the code, run `gulp` and check out the visibility of Direct PLUS loans
- Run those functional tests!
## Review
- @marteki @niqjohnson @higs4281 
## Screenshots

![screen shot 2016-02-24 at 5 29 41 pm](https://cloud.githubusercontent.com/assets/1490703/13303304/8233f1a6-db1c-11e5-8985-0b7e49348f23.png)
## Checklist
## Animated GIF

![it-crowd-internet](https://cloud.githubusercontent.com/assets/1490703/13303667/9c3e919e-db1e-11e5-9165-bb78836fe975.gif)
